src/main/java/io/cdap/plugin/http/common/http/HttpClient.java

```diff
@@ -167,10 +167,10 @@ private Header getAuthorizationHeader(AccessToken accessToken) {
   /**
    * This class allows us to send body not only in POST/PUT but also in other requests.
    */
-  private static class HttpRequest extends HttpEntityEnclosingRequestBase {
+  public static class HttpRequest extends HttpEntityEnclosingRequestBase {
     private final String methodName;
 
-    HttpRequest(URI uri, String methodName) {
+    public HttpRequest(URI uri, String methodName) {
       super();
```

Comment:
Move this class outside in a separate file.

Reply:
Moved